src/pages/ServiceList/FiltersAndSorts.ts

```diff
@@ -255,18 +255,6 @@ export const sortServices = (
   services: ServiceListItem[],
   sortField: SortField<ServiceListItem>,
   isAscending: boolean
-): Promise<ServiceListItem[]> => {
-  if (sortField.title === 'Health') {
-    // In the case of health sorting, we may not have all health promises ready yet
-    // So we need to get them all before actually sorting
-    const allHealthPromises: Promise<WithServiceHealth<ServiceListItem>>[] = services.map(item => {
-      return item.healthPromise.then((health): WithServiceHealth<ServiceListItem> => ({ ...item, health }));
-    });
-    return Promise.all(allHealthPromises).then(arr => {
-      return arr.sort(isAscending ? sortField.compare : (a, b) => sortField.compare(b, a));
-    });
-  }
-  // Default case: sorting is done synchronously
-  const sorted = services.sort(isAscending ? sortField.compare : (a, b) => sortField.compare(b, a));
-  return Promise.resolve(sorted);
+): ServiceListItem[] => {
+  return services.sort(isAscending ? sortField.compare : (a, b) => sortField.compare(b, a));
```

Comment:
Not related to this line....

Perhaps change the declaration of the `filterBy` method to not return `Promise<ServiceListItem[]>`? Because I see it can no longer be a Promise.